prow/cluster/jobs/istio/envoy/istio.envoy.release-1.4.gen.yaml

```diff
@@ -0,0 +1,31 @@
+# THIS FILE IS AUTOGENERATED. See prow/config/README.md
+presubmits:
+  istio/envoy:
+  - always_run: true
+    annotations:
+      testgrid-dashboards: istio_release-1.4_envoy
+    branches:
+    - ^release-1.4$
+    decorate: true
+    name: test-asan_envoy_release-1.4
+    path_alias: istio.io/envoy
+    spec:
+      containers:
+      - command:
+        - ./ci/do_circle_ci.sh
+        - bazel.asan
+        - //test/extensions/access_loggers/wasm/... //test/extensions/filters/http/wasm/...
```

Comment:
We definitely want to execute all tests (just call `./ci/do_circle_ci.sh bazel.asan`), and not only Wasm tests (this is what we do in [envoyproxy/envoy-wasm](https://github.com/envoyproxy/envoy-wasm), but that's because we only modify that part of the codebase there).

Reply:
Updated to include all tests. Also marked the job(s) as **optional** for now since they are not passing in the canary run: https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_envoy/152/test_envoy/11.